src/model/kaniBinaryRunner.ts

```diff
@@ -181,13 +177,38 @@ async function captureFailedChecksForProof(
 // Run a command and capture the command line output into a string
 async function catchOutput(command: string, cargoKaniMode: boolean = false): Promise<number> {
 	const process = await execLog(command);
+	const process2 = await execLog2(command);
+	console.log(process2);
 	return process;
 }
 
+// exectute the command as a command line argument
+async function execLog2(command: string, cargoKaniMode: boolean = false): Promise<any> {
```

Comment:
Could we have a more descriptive name for this function? I can't tell what is different from `execLog`. If the difference is that it calls `execAsyncFile` rather than `execAsync`, then I suggest the function should have the word `File` somewhere in the name.


Same for `process` and `process2` above, I can't tell the difference.

If `execLog` is no longer used, I suggest deleting it.